FIXED double "need to" on "If statements" page

beerlincur opened the pull request:
just updated the code, npm run build was not run

anton-trunov reviewed (dismissed):
Thanks for the fix! If you can rebase your PR, I'm happy to merge it

beerlincur commented:
> Thanks for the fix! If you can rebase your PR, I'm happy to merge it

hello! trying to rebase onto your main branch. facing numerous merge conflicts in build-result files. trying to Accept Your changes, but it shows errors and conflicts about some deleted files anyway.

anton-trunov commented:
A workaround here would be to just have the changes in the markdown files. I'll regenerate the rest of the files in a separate PR

beerlincur commented:
> A workaround here would be to just have the changes in the markdown files. I'll regenerate the rest of the files in a separate PR

reverted my [builded] commit and rebased it. could you, please, take a look at result?

anton-trunov commented:
Looks like something went wrong with the rebase:
<img width="216" alt="image" src="https://github.com/tact-lang/tact-by-example/assets/2316541/03afab46-a0cf-4c69-95b2-1fc9f054a8e2">
